lolFight2/lolFight2/ChooseController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ChooseController.swift
+//  lolFight2
+//
+//  Created by Larissa Diniz  on 22/01/20.
+//  Copyright © 2020 Larissa Diniz . All rights reserved.
+//
+
+import UIKit
+
+class ChooseController: UIViewController {
+
+    @IBOutlet weak var champion1Image: UIImageView!
+    @IBOutlet weak var champion2Image: UIImageView!
+    
+    var champion1Winner : Bool = false
+    var boot : Champion?
+    var myChampion : Champion?
+    
+    let lux = Mage(name: "Lux", life: 100, kills: 0, deaths: 0)
+    let garen = Tank(name: "Garen", life: 100, kills: 0, deaths: 0)
+    let missFortune = Shooter(name: "Miss Fortune", life: 100, kills: 0, deaths: 0)
+    let taric = Support(name: "Taric", life: 100, kills: 0, deaths: 0)
+    
+    @IBAction func Lux(_ sender: UIButton) {
+        champion1Image.image = UIImage(named: "lux3")
+        myChampion = lux
+    }
+    
+    @IBAction func Taric(_ sender: UIButton) {
+        champion1Image.image = UIImage(named: "taric")
+        myChampion = taric
+    }
+    
+    @IBAction func Garen(_ sender: UIButton) {
+        champion1Image.image = UIImage(named: "garen4")
+        myChampion = garen
+    }
+    
+    @IBAction func Missfortune(_ sender: UIButton) {
+        champion1Image.image = UIImage(named: "missFortune2")
+        myChampion = missFortune
+    }
+    
+    @IBAction func Attack(_ sender: UIButton) {
+        let fight1 = Fight.init(champion1: myChampion!, champion2: boot!)
```

Comment:
I suggest you to use the `guard let` here too, to check if `myChampion` and `boot` exists

```swift
guard let champion = myChampion, let boot = boot else { return }

let fight1 = Fight(champion1: champion, champion2: boot)
...
```